types/trouter/index.d.ts

```diff
@@ -0,0 +1,78 @@
+// Type definitions for trouter 3.0
+// Project: https://github.com/lukeed/trouter
+// Definitions by: Markus Lanz <https://github.com/stahlstift>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.3
+
+export interface FindResult<T> {
```

Comment:
You want to put `FindResult`, `HTTPMethod`, and `FindReturn` into a `namespace Trouter` that merges with the class.

Reply:
I've no idea what this means, but it was done originally with contention: https://github.com/DefinitelyTyped/DefinitelyTyped/pull/33557/commits/e83a7604d0a8f4a4d51a1bba9296c8774cf7992c#r261887776

Reply:
![image](https://user-images.githubusercontent.com/2932786/54160747-9d8c6c00-440d-11e9-8f7a-641c2aec9e33.png)
It was thought unneeded because of the incorrect default export - witht hat gone, since it's now using `export =` to match runtime behavior, the namespace is actually required.

Reply:
Okay, thanks for the explanation 👍 I'll wrap it as before. However, does this mean that users won't be able to import the `HTTPMethod` & co?

Reply:
You'll still be able to. `import {HTTPMethod} from "trouter"` will work fine - you'll just need to `import Trouter = require("trouter")` for the class. `Trouter.HTTPMethod` will also work on the latter.

Reply:
Great, thanks for your help and patience :)

Reply:
Sorry about this, I've been reviewing quite a lot of packages the last couple of days and might've mixed up exports in this case.

Reply:
All good! I was confused to begin with 😉